myriad-scheduler/src/main/java/com/ebay/myriad/Main.java

```diff
@@ -190,6 +212,31 @@ private void startNMInstances(Injector injector) {
       }
     }
 
+    /**
+     * Create NMProfile for any configured service
+     * @param cfg 
+     * @param injector
+     */
+    private void initJavaBasedTaskConfiguration(MyriadConfiguration cfg, Injector injector) {
+      LOGGER.info("Initializing JavaBasedTaskConfiguration");
+      ServiceProfileManager profileManager = injector.getInstance(ServiceProfileManager.class);
+      TaskConstraintsManager taskConstraintsManager = injector.getInstance(TaskConstraintsManager.class);
+
+      Map<String, ServiceConfiguration> auxServicesConfigs = 
```

Comment:
Do we still have a notion of aux services? If not, these can be renamed to just serviceConfigs.